src/index.js

```diff
@@ -1,23 +1,41 @@
-import _ from 'lodash';
 import './style.css';
-import printMe from './print.js';
 
- function component() {
-   const element = document.createElement('div');
-
-  const btn = document.createElement('button');
-
-   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-
-  btn.innerHTML = 'Click me and check the console!';
-
-  btn.onclick = printMe;
-
-
-  element.appendChild(btn);
-
-
-   return element;
- }
-
- document.body.appendChild(component());
\ No newline at end of file
+const tasksArray = [];
+
+function createTask(obj) {
+  let myindex;
+  if (tasksArray.length === 0) {
+    myindex = 0;
+  } else {
+    myindex = tasksArray.length;
+  }
+
+  const task = {
+    description: obj,
+    completed: false,
+    index: myindex,
+  };
+  tasksArray.push(task);
+
+  const contenedor = document.querySelector('#contenedorTasks');
+  contenedor.innerHTML += `<li>
+  <div class="taskCont">
+    <input type="checkbox">
+    <div class="task">${task.description}</div>
+  </div>
+  <div class="icon">&#8942</div>
+</li>
+`;
+  const inputText = document.querySelector('#inputNewTask');
+  inputText.value = '';
+}
+
+window.onload = () => {
+  // const button = document.querySelector('.clearAll');
+  // button.onclick = function name() { sayHi(); };
```

Comment:
- [ ] **Optional:** Kindly remove unused code. This makes your code clean and professional.